lib/Chat/SystemMessage/Parser.php

```diff
@@ -55,40 +59,93 @@ public function parseMessage(IComment $comment, string $displayName): array {
 			$parsedMessage = $this->l->t('{actor} left the call');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You left the call');
+			}
 		} else if ($message === 'call_ended') {
 			list($parsedMessage, $parsedParameters) = $this->parseCall($parameters);
 		} else if ($message === 'guests_allowed') {
 			$parsedMessage = $this->l->t('{actor} allowed guests in the conversation');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You allowed guests in the conversation');
+			}
 		} else if ($message === 'guests_disallowed') {
 			$parsedMessage = $this->l->t('{actor} disallowed guests in the conversation');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You disallowed guests in the conversation');
+			}
 		} else if ($message === 'password_set') {
 			$parsedMessage = $this->l->t('{actor} set a password for the conversation');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You set a password for the conversation');
+			}
 		} else if ($message === 'password_removed') {
 			$parsedMessage = $this->l->t('{actor} removed the password for the conversation');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You removed the password for the conversation');
+			}
 		} else if ($message === 'user_added') {
 			$parsedParameters['user'] = $this->getUser($parameters['user']);
 			$parsedMessage = $this->l->t('{actor} added {user} to the conversation');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You added {user} to the conversation');
+			} else if ($currentUser instanceof IUser && $currentUser->getUID() === $parsedParameters['user']['id']) {
+				$parsedMessage = $this->l->t('{actor} added you to the conversation');
+			}
 		} else if ($message === 'user_removed') {
 			$parsedParameters['user'] = $this->getUser($parameters['user']);
-			$parsedMessage = $this->l->t('{actor} removed {user} from the conversation');
+			if ($parsedParameters['user']['id'] === $parsedParameters['actor']['id']) {
+				$parsedMessage = $this->l->t('{actor} left the conversation');
+			} else {
+				$parsedMessage = $this->l->t('{actor} removed {user} from the conversation');
+				if ($currentUserIsActor) {
+					$parsedMessage = $this->l->t('You removed {user} from the conversation');
+				} else if ($currentUser instanceof IUser && $currentUser->getUID() === $parsedParameters['user']['id']) {
+					$parsedMessage = $this->l->t('{actor} removed you from the conversation');
+				}
+			}
 		} else if ($message === 'moderator_promoted') {
 			$parsedParameters['user'] = $this->getUser($parameters['user']);
 			$parsedMessage = $this->l->t('{actor} promoted {user} to moderator');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You promoted {user} to moderator');
+			} else if ($currentUser instanceof IUser && $currentUser->getUID() === $parsedParameters['user']['id']) {
+				$parsedMessage = $this->l->t('{actor} promoted you from moderator');
```

Comment:
This should be "promoted you to moderator"

Reply:
@Ivansss is right.

Reply:
fixed